src/layers/microscopy-viewer-layer/microscopy-viewer-layer.js

```diff
@@ -49,12 +49,14 @@ export class MicroscopyViewerLayer extends BaseTileLayer {
     })
     var diff = 6 - orderedSliderValues.length
     for (var i = 0; i < diff; i++) {
-      orderedSliderValues.push(65535);
+      orderedSliderValues.push([0,65535]);
     }
     var diff = 6 - orderedColorValues.length
     for (var j = 0; j < diff; j++) {
       orderedColorValues.push([0,0,0]);
     }
+    // flatten for use on shaders
+    var flatSliderValues = [].concat.apply([], orderedSliderValues)
```

Comment:
- Prefer const to var. (Linting!)
- Prefer to have definitions as close as possible to the thing that uses them. (Though maybe all these are used together, so it couldn't be closer without something else being farther?)

Reply:
We will prettier/eslint on this in one pass later